tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+import logging
+import time
+
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.framework.testlib import ignore_leftovers, ManageTest, tier4
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources import pod as pod_helpers
+from ocs_ci.ocs.resources import storage_cluster
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs.node import get_typed_nodes, wait_for_nodes_status
+
+
+@pytest.mark.parametrize(
+    argnames=["node_type", "num_of_nodes", "workload_storageutilization_rbd"],
+    argvalues=[
+        pytest.param(
+            *['worker', 1, (0.11, True, 120)],
+            marks=pytest.mark.polarion_id("OCS-1313")
+        ),
+    ],
+    indirect=["workload_storageutilization_rbd"],
+)
```

Comment:
there is only a single set of params here. Is pytest.mark.parametrize even needed?

Reply:
I think it will be easier to write it that way. And also we need it for the pytest indirect parameters, as I can see in the docs: https://docs.pytest.org/en/latest/example/parametrize.html#apply-indirect-on-particular-arguments

Reply:
ok